test/verify/check-storage-smart

```diff
@@ -136,6 +136,33 @@ class TestStorageSmart(storagelib.StorageSmartCase):
         b.wait_visible(self.card("Media drive"))
         b.wait_not_present(self.card("Device health (SMART)"))
 
+    def testSmartOverview(self):
```

Comment:
These are expensive destructive tests -- can you please fold these into the existing testSmart() and just switch back and forth between Overview and Storage pages?

Reply:
ack!, will do